packages/waku/src/lib/builder/serve-cloudflare.ts

```diff
@@ -1,41 +1,75 @@
 import { Hono } from 'hono';
+import { unstable_getPlatformObject } from 'waku/server';
```

Comment:
It might not change anything real, but for consistency. (Will be refactored anyway.)
```suggestion
import { unstable_getPlatformObject } from '../../server.js';
```